com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ExtenderUtil.java

```diff
@@ -757,24 +758,10 @@ public static boolean isAndroidAssetDirectory(Project project, String path) {
         return false;
     }
 
-    public static byte[] createSha1(File file) throws Exception  {
```

Comment:
Another place where we consume a file and do an operation on the bytes. And we're not wrapping into a BufferedInputStream. Moved to FileUtil.